store-api-server/src/user/service/dipdup.ts

```diff
@@ -0,0 +1,81 @@
+import axios from 'axios'
+export async function getFromDipdup(walletAddress: string) {
+  const axiosTokenMetadataResponse = (await axios({
+    url: `http://${process.env['HASURA_URL']}:8080/v1/graphql`,
+    method: 'post',
+    headers: { 'x-hasura-admin-secret': 'changeme' },
+    data: {
+      query: `query {
+        create_token {
+          contract
+          counter
+          hash
+          id
+          metadata
+          metadata_uri
+          timestamp
+          token_id
+          count
+        }
+      }`
+    }
+  }))?.data
+  if (axiosTokenMetadataResponse.data.errors?.length) {
+    throw new Error(
+      `error from hasura on create-token request: ${axiosTokenMetadataResponse.data.errors[0].message}`
+    )
+  }
+  const axiosResponse = (await axios({
+    url: `http://${process.env['HASURA_URL']}:8080/api/rest/get_user_by_address?address=${walletAddress}`,
+    method: 'get',
+    headers: { 'x-hasura-admin-secret': 'changeme' },
+  }))?.data
+  if (axiosResponse.errors?.length) {
+    throw new Error(
+      `error from hasura on get_user_by_address request: ${axiosResponse.errors[0].message}`
+    )
+  }
+
+  const IPFS_GATEWAY = `https://cloudflare-ipfs.com/ipfs/`
```

Comment:
Are we confident in using this? Or should this gateway be imported from Secrets in case we want to change to the Trili Pinata gateway?